CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+Creating Issues
+===============
+
+## :beetle: Reporting bugs
+
+reporting bugs
+
+
+## :sparkles: Requesting features
+
+requesting features
+
+
+Contributing Code to MockBukkit
+===============================
+
+## :toolbox: Software requirements
+
+To get started contributing to MockBukkit, you will need a few things.
+- [Git](https://git-scm.com/downloads)
+- Java 17 or newer ([Windows Download](https://download.oracle.com/java/17/archive/jdk-17.0.3.1_windows-x64_bin.exe)).
+
+
+## :wrench: Setting up the project
+
+The first step in contributing to MockBukkit, is to create a fork of it. You can do this by clicking the `Fork` button in the top-right of the GitHub repository.
+Once your fork is created, you will need to clone it to your local system.
+
+To do this, on GitHub, click on the green `Code` button on the homepage of your fork, then copy the provided URL.
+Next, if on Windows open up `Git Bash` and navigate to where you want to clone the repo.
+If on Linux, simply navigate to where you want to clone the repo.
+
+Once you're where you want to clone the repo, run `git clone <url>` with `<url>` being the URL you just copied, omitting the `<>`.
+
+When the previous command is finished executing, you will need to create your own branch.
+To do this, run `git checkout -b <branch name>` with `<branch name>` being the name of your new branch.
+There cannot be two branches with the same name. The recommended naming scheme for branch is `feature/my-feature` for adding something new, and `fix/my-fix` for fixing a bug.
+`my-feature`/ `my-fix` should obviously be a condensed name of what you're adding/ fixing.
+
+Now you can get to coding!
+
+
+## :black_nib: Code formatting
+
+All pull requests should adhere to our code style.
+If you're using a modern IDE like Intellij, we have a `.editorconfig` file which will tell it how code should be formatted.
+This allows you to use auto code formatting without changing more than you modified.
+If for whatever reason your IDE does not follow our style, please, do **not** commit changes that unnecessarily modify code.
```

Comment:
This would sound ***much*** friendlier in my opinion.

```suggestion
Every larger project has a certain formatting style which dictates how code is formatted.
If you're using a modern IDE like Intellij, we have a `.editorconfig` file within the project which will automatically tell your IDE how our code should be formatted.
This allows you to use auto code formatting without disrupting the project's code style.

We would appreciate it if all pull requests adhere to this project's code style.
If for whatever reason your IDE does not support this or enforces a different code style, try to manually keep the style as consistent as possible in regards to the surrounding code.
But don't worry, we won't reject your hard work just because of a misaligned bracket.
We will help you out with formatting mistakes by suggesting changes to your PR!
```

Reply:
Indeed that sounds much better. Thank you!